index.js

```diff
@@ -114,6 +114,7 @@ exports.init = function (sbot, config) {
   })
 
   function onClose () {
+    // TODO: remove this, no one seems to depend on it
```

Comment:
Yes, this one is a bit strange. The idea with EBT is that you have a long running connection. So it never really finished unless you actually disconnects somehow.